plugins/graphql-voyager/README.md

```diff
@@ -0,0 +1,75 @@
+# graphql-voyager
+
+Welcome to the graphql-voyager plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Getting started
```

Comment:
Can we include a screenshot here? It really helps people see what the plugin does.

Also, I'm not sure including the "created through the Backstage CLI" is needed